composer.json

```diff
@@ -28,7 +28,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.5.x-dev"
+            "dev-master": "2.6.x-dev"
```

Comment:
This change is both unrelated, and unneeded. Composer only respects the alias on the actual master branch.